acceptance/bundle/generate/app_not_yet_deployed/test.toml

```diff
@@ -0,0 +1,2 @@
+Local = true
+Cloud = false
```

Comment:
This is default, can be omitted (especially given that we have out.test.toml).

Reply:
yes, the intention is not to run this on cloud explicitly